docs/reference/python-api.rst

```diff
@@ -1,10 +1,10 @@
 .. _python-api:
```

Comment:
Can you also:
- rename this file in git
- include and file redirect
- rename this `python-api` reference?

Reply:
done